src/PrestaShopBundle/Resources/views/Admin/Improve/International/Localization/Blocks/configuration.html.twig

```diff
@@ -0,0 +1,88 @@
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% trans_default_domain 'Admin.International.Feature' %}
+
+{% block localization_configuration %}
+  <div class="card">
+    <h3 class="card-header">
+      <i class="material-icons">settings</i> {{ 'Configuration'|trans({}, 'Admin.Global') }}
+    </h3>
+
+    <div class="card-block">
+      <div class="card-text">
+        <div class="form-group">
+          {{ ps.label_with_help(('Default language'|trans), ('The default language used in your shop.'|trans({}, 'Admin.International.Help'))) }}
+          {{ form_errors(configurationForm.default_language) }}
+          {{ form_widget(configurationForm.default_language) }}
+        </div>
+
+        <div class="form-group">
+          <label class="form-control-label">
+            {{ 'Set language from browser'|trans }}
+          </label>
+          {{ form_errors(configurationForm.detect_language_from_browser) }}
+          {{ form_widget(configurationForm.detect_language_from_browser) }}
+          <small class="form-text">{{ 'Set browser language as default language'|trans({}, 'Admin.International.Help') }}</small>
+        </div>
+
+        <div class="form-group">
+          {{ ps.label_with_help(('Default country'|trans), ('The default country used in your shop.'|trans({}, 'Admin.International.Help'))) }}
+          {{ form_errors(configurationForm.default_country) }}
+          {{ form_widget(configurationForm.default_country) }}
+        </div>
+
+        <div class="form-group">
+          <label class="form-control-label">
+            {{ 'Set default country from browser language'|trans }}
+          </label>
+          {{ form_errors(configurationForm.detect_country_from_browser) }}
+          {{ form_widget(configurationForm.detect_country_from_browser) }}
+          <small class="form-text">{{ 'Set country corresponding to browser language'|trans({}, 'Admin.International.Help') }}</small>
+        </div>
+
+        <div class="form-group">
+          {{ ps.label_with_help(('Default currency'|trans), ('The default currency used in your shop.'|trans({}, 'Admin.Shopparameters.Help'))) }}
```

Comment:
_The default currency used in your shop._ must be attached to the Admin.International.Help domains.

Reply:
yup, thanks. :)

Reply:
You're welcome, Šarūnas!